client/src/components/App/index.js

```diff
@@ -1,12 +1,134 @@
 import React from 'react';
+import { Switch, Route, Redirect } from 'react-router-dom';
+import axios from 'axios';
+
+import AppBar from '../common/AppBar';
+import Limitation from '../common/limitation';
+import Error404 from '../pages/common/errors/Error-404';
+import Error500 from '../pages/common/errors/Error-500';
+import Home from '../pages/common/Home';
+import Login from '../pages/common/Login';
+import Signup from '../pages/application/Signup';
+import Availability from '../pages/application/Availability';
+import Tasks from '../pages/application/Tasks';
+import Project from '../pages/application/Project';
+import Submit from '../pages/application/Submit';
+import Profile from '../pages/application/Profile';
+import Dashboard from '../pages/admin/Dashboard';
+import Opened from '../pages/admin/Opened';
+import SubmittedAll from '../pages/admin/Submitted_all';
+import SubmittedId from '../pages/admin/Submitted_Id';
+import Completed from '../pages/admin/Completed';
 import './index.css';
 
-function App() {
-  return (
-    <div>
-      <h1>Welcome in Applicants System</h1>
-    </div>
-  );
-}
+export default class App extends React.Component {
+  state = {
+    user: false,
+    admin: false,
+    loading: true,
+  };
+
+  componentDidMount() {
+    axios
+      .get('/api/v1/isAdmin')
+      .then((result) => {
+        this.setState({ ...result.data });
+      })
+      .catch(() => {
+        this.setState({ admin: false, loading: false });
```

Comment:
admin is already `false` in the state so no need to add it again. maybe its better to be like this
```suggestion
        this.setState({ loading: true });
```